### PR TITLE
Release 0.7.3

### PR DESCRIPTION
**Pull request by ScottDelbecq**
https://github.com/fast-aircraft-design/FAST-OAD_CS25/releases/tag/untagged-6fe712bdcaabb4ecc2c1

### CODE REVIEW

**Comment by ScottDelbecq** [comment]
> Your branch is not based on the HEAD of main, so it does not include #49 😅

Woops! Thanks I missed a failed fetch I guess!

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,14 +2,19 @@
 Changelog
 =========
 
+Version 0.7.23
```
Comment (christophe-david) [inline CHANGELOG.rst:5]:
```suggestion
Version 0.7.3
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
+=============
+Modified:
+
+- Added scaling factors for tuning HTP and VTP areas (#49)
```
Comment (christophe-david) [inline CHANGELOG.rst:9]: Just for sake of consistency, but honestly, I wouldn't have mentioned it if it had been the only point.
```suggestion
- Added scaling factors for tuning HTP and VTP areas. (#49)
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,14 +2,19 @@
...
+
 Version 0.7.2
 =============
 Modified:
 
 - Python 3.12 is now supported. (#48)
 - Python 3.8, that reached end of life, is no longer supported. (#48)
 
-
```
Comment (christophe-david) [inline CHANGELOG.rst:12]: And well done for consistency 😃

**Review by christophe-david — CHANGES_REQUESTED** [review]
Some minor changes needed in the change log, and also, please rename the release as `v0.7.3`.

File: CHANGELOG.rst (hunk 3 above)
Comment (ScottDelbecq) [reply CHANGELOG.rst:12]: haha thx

**Review by christophe-david — CHANGES_REQUESTED** [review]
Ok for the change log, but please, rename the [release](https://github.com/fast-aircraft-design/FAST-OAD_CS25/releases/tag/untagged-1dcca1a555921365cc22) as `v0.7.3`, consistently with previous releases.

**Pull request merged**